docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -88,21 +56,21 @@ _(these items were already deprecated and did nothing)_
 - `<option name="giveUnitControl" value="true"/>`
 - `<option name="giveUnitControl" value="false"/>`
 
-### Changes of map (properties) file items:
+## map.properties Changes
 
 Within every "map" properties file, inside the main folder of the map, change:
 
 - `color.Impassible=` >> `color.Impassable=`
 
-### Changes of sounds (properties) file items:
+## Sound file changes
 
-Within every "sounds" properties file, inside the main folder of the map, change:
+Change all sounds inside the "sounds" main folder of the map from "wav" to "mp3" format.
+128Kbps and 168Kbps are the preferred bit rates.
 
-- `.wav` >> `.mp3`
 
-### Changes of non-code items:
+## More XML Changes
 
-Change all sounds inside the "sounds" main folder of the map from "wav" to "mp3" format.
+### Changes of non-code items:
```

Comment:
This line used to be before the information about changing the format of the sound files. The "non-code" items mentioned were only the needed change of substituting all "wav" files with "mp3" ones, which is now detailed before this line. I believe everything else are code changes. So this line does not make any sense to me (I don't get what are these "non-code" items, now). Moreover, since a section break follows, it seems to me that this is even saying the "non-code items" part is empty.